pkgs/tools/misc/vdirsyncer/stable.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, python3Packages, fetchpatch }:
+
+# Packaging documentation at:
+# https://github.com/pimutils/vdirsyncer/blob/dcf5f701b7b5c21a8f4e8c80243db3e0baff1313/docs/packaging.rst
+python3Packages.buildPythonApplication rec {
+  version = "0.16.7";
+  pname = "vdirsyncer";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "6c9bcfb9bcb01246c83ba6f8551cf54c58af3323210755485fc23bb7848512ef";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click click-log click-threading
+    requests_toolbelt
+    requests
+    requests_oauthlib # required for google oauth sync
+    atomicwrites
+    milksnake
+    shippai
```

Comment:
I can't find the last two in https://github.com/pimutils/vdirsyncer/blob/0.16.7/setup.py.